src/api/controllers/themeController.ts

```diff
@@ -37,9 +38,10 @@ const getThemes = async (req: Request, res: Response) => {
 			limit,
 			offset
 		});
-		res.json(themes);
+		sendSuccessResponse(res, 200, themes, "Themes fetched successfully.");
 	} catch (error) {
-		res.status(500).json({ error: "Failed to fetch themes" });
+		Logger.error("Error fetching themes:", error);
+		sendErrorResponse(res, 500, "Failed to fetch themes.");
 	}
```

Comment:
This is a general comment regarding controllers.

I do not see any global error handler. Can it be used here so that we do not have to provide `catch` clause for every controller route? It looks like Express does have a way to do so from my quick search https://medium.com/@mohsinogen/simplified-guide-to-setting-up-a-global-error-handler-in-express-js-daf8dd640b69

Reply:
I took a look at the global error handler, and I'm convinced it is pretty neat to have 😝 I can see the value in having it, but in the interest of unblocking other issues, let's refactor this separately.